corehq/apps/reports/static/reports/js/scheduled_reports_list.js

```diff
@@ -0,0 +1,185 @@
+/*
+
+    ScheduledReportModel - model representing a single row on the list / a single report filter file
+    ScheduleReportPanelModel - model representing a whole table ('My Scheduled Reports' and 'Other Scheduled Reports')
+    ScheduledReportListModel - model representing the whole page (list of reports, actions)
+
+*/
+
+hqDefine("reports/js/scheduled_reports_list", [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/components.ko', // pagination & feedback widget
+
+], function (
+    $,
+    ko,
+    _,
+) {
+
+    var scheduledReportModel = function (data, isOwner, isAdmin) {
+
+        var self = ko.mapping.fromJS(data);
+
+        self.configMany = ko.computed(function() {
```

Comment:
`ko.computed` is needed when referencing observables in the computation. However, it looks like this references a static initial value. Am I reading this correctly?

Reply:
the utility for `ko.computed` with observables is that it's re-computed when the value of the observable is updated. however, with a static value, the value never changes, so such a variable can just be set at initialization.